data/gibi.py

```diff
@@ -0,0 +1,119 @@
+"""This module provides a custom PyTorch GeoDataset for working with vector data.
```

Comment:
I don't think this actually works since GIBI isn't polygons, no? What is the rasterization rasterizing? If it does work but the dots are currently in the wrong place please note that here in the description. If it doesn't work note what is missing.